tests/test_statistics.py

```diff
@@ -4,10 +4,13 @@
 """
 from __future__ import absolute_import
 
-from .statistics import nan_percentile, argpercentile, axisindex
+from datacube_stats.statistics import nan_percentile, argpercentile, axisindex
```

Comment:
While not urgent, can we also test `datacube_stats.statistics.section_by_index` (my version of `axisindex` after I gave up on understanding the code) which should behave identically (on all valid inputs)?